sts-e2e-tests/src/test/java/de/adorsys/sts/tests/e2e/tokenexchange/TokenExchangeControllerJpaTest.java

```diff
@@ -55,6 +55,6 @@ void tokenExchangeTest() {
                 .andExpect(status().isOk())
                 .andExpect(jsonPath("$.issued_token_type").value("urn:ietf:params:oauth:token-type:access_token"))
                 .andExpect(jsonPath("$.token_type").value("Bearer"))
-                .andExpect(jsonPath("$.expires_in").value("84580358"));
+                .andExpect(jsonPath("$.expires_in").value("315576000"));
```

Comment:
This change is not necessary when injecting Clock (no changes needed in the test)

Reply:
```suggestion
                .andExpect(jsonPath("$.expires_in").value("84580358"));
```